test/appengine/conftest.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import sys
+
+# Import py.test hooks and fixtures for App Engine
+from gcp_devrel.testing.appengine import (
+    pytest_configure,
+    pytest_runtest_call,
+    testbed)
+import pytest
+import six
+
+(pytest_configure)
+(pytest_runtest_call)
+(testbed)
```

Comment:
Hah, this is copy-pasta because the linter on the project this is from gets angry at unused imports. I switched this to use `__all__` instead, which you may find easy to swallow.